github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-postsubmits.yaml

```diff
@@ -10,18 +10,18 @@ postsubmits:
       labels:
         preset-dind-enabled: "true"
         preset-docker-mirror-proxy: "true"
-        preset-project-infra-kubevirtci-docker-credential: "true"
+        preset-kubevirtci-quay-credential: "true"
       spec:
         nodeSelector:
           type: vm
         containers:
-          - image: kubevirtci/bootstrap:v20201119-a5880e0
+          - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
```

Comment:
is it possible to use a floating tag for `quay.io/kubevirtci/bootstrap` image, (e.g. "`latest`"), instead of manually updating it in the repo from time to time?

Reply:
All project-infra uses a frozen boostrap image, 
currently there is no "latest" on https://quay.io/repository/kubevirtci/bootstrap?tag=latest&tab=tags
so we can't now, and i don't think it should be like that, because it should be controlled,
i.e if new image has errors, we don't want all repos to be hurt from that, but to update it from time to time when needed,
the bootstrap image isn't updated so often i think,

@fgimenez wdyt ?

btw 
i can remove it from this PR as its not related to the main change which is to publish to quay if needed.

Reply:
then maybe not latest, but you can introduce a `stable` tag, that you update it with proven images that are working without errors.
it will be more scalable, not needing to update the tag in each repo that uses kubevirt-ci

Reply:
Its not related to kubevirtci, its just under the same repo, its the bootstrap, where the prow job runs.
I will remove it as part of this PR, as its not related, and we can discuss it deeper please if its ok.